src/System.IO/tests/TextWriter/TextWriterTests.cs

```diff
@@ -0,0 +1,598 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public partial class TextWriterTests
+    {
+        protected static string strfmtOneObject = "Object is {0}";
+        protected static string strfmtTwoObjects = $"Object are '{0}', {secondObject}";
+        protected static string strfmtThreeObjects = $"Objects are {0}, {secondObject}, {thirdObject}";
+        protected static string strfmtMultipleObjects = "Multiple Objects are: {0}, {1}, {2}";
+
+        protected static object firstObject = (object)1;
+        protected static object secondObject = (object)"[second object]";
+        protected static object thirdObject = (object)"<third object>";
+        protected static object[] multipleObjects = new object[] { firstObject, secondObject, thirdObject };
```

Comment:
Nit: all of these statics should begin with a capital letter.

Reply:
done!